src/__integrationtests__/Balance.spec.ts

```diff
@@ -65,7 +70,8 @@ describe('when there is a dev chain with a faucet', () => {
     const funny = jest.fn()
     listenToBalanceChanges(ident.address, funny)
     const balanceBefore = await getBalance(faucet.address)
-    await makeTransfer(faucet, ident.address, MIN_TRANSACTION)
+    const tx = await makeTransfer(faucet, ident.address, MIN_TRANSACTION)
```

Comment:
This is better already. It is more verbose. 

Rolling this out will make things clearer when we action a `tx` to be written on the chain.

Reply:
yesss that was the point. now noone can say they didn't know it would cost tokens. In theory we should also be able to check transaction costs beforehand